src/map/battle.c

```diff
@@ -7587,6 +7585,13 @@ static void battle_adjust_conf(void)
 	}
 #endif
 
+#if PACKETVER < 20161207
+	if( battle_config.feature_enable_multi_crit ) {
```

Comment:
please use ``if`` in this format ``if (battle_config.feature_enable_multi_crit) {``